libs/dh/admin/feature-user-management/src/lib/drawer/roles/dh-role-drawer.component.html

```diff
@@ -16,8 +16,20 @@
 -->
 <ng-container *transloco="let t; read: 'admin.userManagement.drawer'">
   <watt-drawer #drawer size="large" (closed)="onClose()" [loading]="isLoading$ | push">
-    <watt-drawer-topbar></watt-drawer-topbar>
-
+    <watt-drawer-topbar *ngIf="drawer.isOpen">
+      <ng-container *rxLet="userRoleWithPermissions$ as userRoleWithPermissions">
+        <dh-role-status [status]="userRoleWithPermissions.status"></dh-role-status>
+      </ng-container>
+    </watt-drawer-topbar>
+    <watt-drawer-actions-content *ngIf="drawer.isOpen">
+      <ng-container *rxLet="userRoleWithPermissions$ as userRoleWithPermissions">
+        <div class="role-name__grid">
+          <h2 class="role-name__headline">
+            {{ userRoleWithPermissions.name }}
+          </h2>
+        </div>
```

Comment:
suggestion: the extra element was necessary before, when there was a status badge under the heading, but the badge is not here anymore. The extra element and styling can be removed.